index.d.ts

```diff
@@ -0,0 +1,138 @@
+// Type definitions for looks-same 4.0
+// Project: https://github.com/gemini-testing/looks-same/releases
+// Definitions by: xcatliu <https://github.com/xcatliu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node"/>
+
+type LooksSameCallback = (error: Error | null, equal: boolean) => any;

```

Comment:
why returned type is `any` and not a `void`?
Documentation - http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html (Callback Types)